src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+if [ "$ARCH" == "arm" ]; then
+  BASE_IMAGE=debian:jessie
+  ARM_BASE_IMAGE=armhf/debian:jessie
```

Comment:
I just had a look at `resin/rpi-raspbian` and it doesn't seem to be different. It's also an `armhf` image, so it will probably work on RPi1's with hard-float OS.